lib/logsink.js

```diff
@@ -43,7 +42,7 @@ function timestamp () {
   if (!timeZone) {
     date = new Date(date.valueOf() + date.getTimezoneOffset() * 60000);
   }
-  return date.toFormat("YYYY-MM-DD HH24:MI:SS:LL");
+  return dateformat(date, "yyyy-mm-dd H:M:ss:l");
```

Comment:
It's 1-24. I just pushed another commit because I noticed that `H` doesn't include leading zero and neither does `M` so I replaced it with `HH:MM`.